AdrenaFoundation/frontend
# Improve leaderboard + add CTA to create user profile

- update trading view to latest
- update Ranked self position display
- add "create profile" CTA
- add total user count (would be cool to see "user live" instead, like users trading right 
now)
+ update trading view

TODO
- mobile responssive on that page is fuck, pls help @adrena-tyro king
